src/nets/e2e_asr_attctc_th.py

```diff
@@ -356,6 +357,7 @@ def recognize(self, x, recog_args, char_list, rnnlm=None):
 
         # 1. encoder
         # make a utt list (1) to use the same interface for encoder
+        h = h.contiguous()
```

Comment:
Is it required?

Reply:
This is required when we use VGG layers initially..